pkg/cmd/admin/router/router.go

```diff
@@ -287,6 +287,7 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 							Spec: kapi.PodSpec{
 								ServiceAccount: cfg.ServiceAccount,
 								NodeSelector:   nodeSelector,
+								HostNetwork: true,
```

Comment:
Stat port can be made a unix socket too. Or we can start using multiple interfaces. Though we should eventually pursue this option, I feel cookies is a safer option for a last minute bug fix.